Azure/azure-cli
# [AKS] `az aks nodepool delete-machines`: Add support to delete specific machines in an agent pool

**Related command**
az aks nodepool delete-machines
delete machines API was added to [2024-07-01](https://github.com/Azure/azure-rest-api-specs/pull/30318)

**Description**
We want to add the new feature delete-machines in GA cli.

**Testing Guide**
Use the command `az aks nodepool delete-machines` to send the request to aks. Use the command `az aks machine list` to verify the results.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
